src/content/glossary.tsx

```diff
@@ -0,0 +1,82 @@
+export const terms = {
+    affinity: {
+        definition:
+            "Measure of the strength of an interaction between two molecules. Generally, the more chemical bonds that form between two molecules, the stronger the affinity of those molecules for each other.",
+        link: "[Learn more]",
```

Comment:
What's the difference between `[Learn more]` and `undefined` in this context? Are these ones supposed to be links later?

Reply:
those originally had links in the design/copy but after I checked, we're not longer have external links for them so I removed it